tests/Paysera/WalletApi/MapperTest.php

```diff
@@ -3,14 +3,26 @@
 namespace Paysera\WalletApi;
 
 use DateTime;
+use Exception;
+use Paysera_WalletApi_Entity_ClientPermissionsToWallet;
 use Paysera_WalletApi_Entity_Location_SearchFilter;
 use Paysera_WalletApi_Entity_Transaction;
 use Paysera_WalletApi_Entity_User_Identity;
+use Paysera_WalletApi_Exception_LogicException;
 use Paysera_WalletApi_Mapper;
 use Paysera_WalletApi_Mapper_IdentityMapper;
+use Paysera_WalletApi_OAuth_Consumer;
 
 class MapperTest extends \PHPUnit_Framework_TestCase
```

Comment:
We have a test for Mapper, as we added new methods into the Client, I think would be also good to add mapper test, so we would be sure it's returning the expected results not random data.

Reply:
how about this? is it done?

Reply:
Yes, I added tests to mapper new tests for encoding and decoding client